dev/sandbox/Dockerfile

```
@@ -4,20 +4,20 @@ FROM ubuntu:24.04
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Install common development tools and runtimes
-# - Node.js: Install v20 (LTS) from NodeSource
+# - Node.js/npm: from Ubuntu's signed package repositories
 # - Core: curl, git, vim, build-essential (gcc, make)
 # - Python: python3, pip
 # - Network: ping, dnsutils
-RUN apt-get update && apt-get install -y curl && \
-    curl -fsSL https://deb.nodesource.com/setup_20.x | bash - && \
-    apt-get install -y \
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)